test/compress/issue-1001.js

```diff
@@ -0,0 +1,8 @@
+parenthesis_strings_in_parenthesis: {
+    input: {
+        ('(');
+        a(')');
+
+    }
+    expect_exact: '"(";a(")");'
```

Comment:
This is actually incorrect. The input contains two statements: a string literal and a function call. The expected result contains a directive and a function call. I would expect either a string literal and a function call or, if the useless string literal were dropped, just a function call.